File: ProgressBarAdv_Scroll/ProgressBarAdv_Scroll_VB/Form1.vb
Imports Syncfusion.Windows.Forms
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Linq
Imports System.Text
Imports System.Threading.Tasks
Imports System.Windows.Forms

Namespace ProgressBar_windows
	Partial Public Class Form1
		Inherits MetroForm

		Public Sub New()
			InitializeComponent()
		End Sub



		Private Sub timer1_Tick(ByVal sender As Object, ByVal e As EventArgs) Handles timer1.Tick
			If Me.progressBarAdv1.Value < 100 Then

				Me.progressBarAdv1.Value += 1

			Else

				Me.progressBarAdv1.Value = 0
			End If
		End Sub

		Private Sub button1_Click(ByVal sender As Object, ByVal e As EventArgs) Handles button1.Click
			Me.timer1.Enabled = True
		End Sub
	End Class
End Namespace

File: ProgressBarAdv_Scroll/ProgressBarAdv_Scroll_C#/Form1.cs
﻿using Syncfusion.Windows.Forms;
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace ProgressBar_windows
{
    public partial class Form1 : MetroForm
    {
        public Form1()
        {
            InitializeComponent();
        }



        private void timer1_Tick(object sender, EventArgs e)
        {
            if (this.progressBarAdv1.Value < 100)

                this.progressBarAdv1.Value++;

            else

                this.progressBarAdv1.Value = 0;
        }

        private void button1_Click(object sender, EventArgs e)
        {
            this.timer1.Enabled = true;
        }
    }
}
